src/fsharp/service/IncrementalBuild.fs

```diff
@@ -979,7 +980,7 @@ type IncrementalBuilder(tcGlobals, frameworkTcImports, nonFrameworkAssemblyInput
             computeStampedFileName cache ctok slot fileInfo (fun slot fileInfo ->
                 let prevSemanticModel =
                     match slot with
-                    | 0 -> initial
+                    | 0 (* first file *) -> initial
```

Comment:
Could a named literal work better than a comment here?

Reply:
We could, but I think it's fine that we have the comment here. At some point, we will do another round of refactoring here, but it will be much easier to do now.